redis/asyncio/sentinel.py

```diff
@@ -117,7 +116,7 @@ def __init__(self, service_name, sentinel_manager, **kwargs):
         self.is_master = kwargs.pop("is_master", True)
         self.check_connection = kwargs.pop("check_connection", False)
         super().__init__(**kwargs)
-        self.connection_kwargs["connection_pool"] = weakref.proxy(self)
+        self.connection_kwargs["connection_pool"] = self
```

Comment:
This sort of change has me wonder why it was a weakref in the first place – probably for a reason?